TeamCode/src/main/java/org/firstinspires/ftc/teamcode/components/ArmSystem.java

```diff
@@ -266,8 +272,15 @@ private int calculateHeight(double pos){
 
     // Must be called every loop
     public boolean runSliderToTarget() {
-        slider.setPower(1.0);
-        return areRoughlyEqual(slider.getCurrentPosition(), slider.getTargetPosition());
+        if (mDirection == ArmDirection.UP ? slider.getCurrentPosition() <  slider.getTargetPosition() :
+                slider.getCurrentPosition() > slider.getTargetPosition()) {
+            slider.setPower(1.0);
+            return false;
+        } else {
+            slider.setPower(0);
```

Comment:
@thort1 : Still curious why braking doesn't work, but this seems to do the trick:
```
            slider.setTargetPosition(slider.getCurrentPosition());
            slider.setMode(DcMotor.RunMode.RUN_TO_POSITION);
```